src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -2071,6 +2071,7 @@ public void SourceItemMissing()
 #else
         [Fact(Skip = "https://github.com/Microsoft/msbuild/issues/297")]
 #endif
+        [PlatformSpecific(Xunit.PlatformID.Windows)]
```

Comment:
Also, the commit this belongs to, enables binary serialization for the mono builds. https://github.com/Microsoft/msbuild/pull/1381/commits/ad50a5356f1be61a6ac2e740a6a1c10618112ec4